Correction of the load_default_certs call condition (#3486)
urllib3/urllib3: changelog/3519.bugfix.rst

> Fixed an issue useless loading system certificates when specifying ``cert_reqs='CERT_NONE'`` in ``HTTPSConnectionPool``

Reviewer: ```suggestion
Prevent loading of system certificates when ``cert_reqs='CERT_NONE'`` is specified for ``HTTPSConnectionPool``.
```
